.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: Release
```

Comment:
Mind linking to https://intuit.github.io/auto/docs/build-platforms/github-actions in the comments here?